tensornetwork/backends/numpy/decompositions.py

```diff
@@ -122,3 +122,28 @@ def rq(
   r = np.reshape(r, list(left_dims) + [center_dim])
   q = np.reshape(q, [center_dim] + list(right_dims))
   return r, q
+
+
+def cholesky(
+    tf: Any,
```

Comment:
Rename `tf` to `np` (the user passes the `numpy` module here)

Reply:
ya